doc/application/application.rst

```diff
@@ -935,155 +935,121 @@ The example below shows a comment line and an override setting
 
-#. After configuring the kernel options for your application, tab to
-   :guilabel:`< Save >` and press :kbd:`Enter`.
+   Pressing :kbd:`ESC` or :kbd:`Q` returns from the information display to the
+   menu display.
 
-   The following dialog opens with the :guilabel:`< Ok >` command selected:
+#. After configuring the kernel options for your application, press
+   :kbd:`Q` to bring up the save-and-quit dialog:
 
    .. image:: figures/app_kernel_conf_2.png
-      :width: 400px
       :align: center
-      :height: 100px
-      :alt: Save Configuration Dialog
-
+      :alt: Save and Quit Dialog
 
-#. Press :kbd:`Enter` to save the kernel configuration options to the default
-   file name; alternatively, type a file name and press :kbd:`Enter`.
+#. Press :kbd:`Y` to save the kernel configuration options to the default
+   filename (:file:`zephyr/.config`).
 
-   Typically, you will save to the default file name unless you are
+   Typically, you will save to the default filename unless you are
    experimenting with various configuration scenarios.
 
-   A :file:`zephyr` directory will have been created in the build
-    directory.
-
    .. note::
 
-      At present, only a :file:`.config` file can be built. If you have saved
-      files with different file names and want to build with one of these,
-      change the file name to :file:`.config`. To keep your original
-      :file:`.config`, rename it to something other than :file:`.config`.
-
-   Kernel configuration files, such as the :file:`.config` file, are saved
-   as hidden files in :file:`zephyr`. To list all your kernel configuration
-   files, enter :command:`ls -a` at the terminal prompt.
+      At present, the configuration file used during building is always
+      :file:`zephyr/.config`. If you have another saved configuration that you
+      want to build with, copy it to :file:`zephyr/.config`. Make sure to back
+      up your original configuration file.
 
-   The following dialog opens, displaying the file name the configuration
-   was saved to.
+      Also note that filenames starting with ``.`` are not listed by ``ls`` by
+      default on Linux and Mac OS X. Use the ``-a`` flag to see them.
 
-   .. image:: figures/app_kernel_conf_3.png
-         :width: 400px
-         :align: center
-         :height: 150px
-         :alt: Saved Configuration Name Dialog
+Finding a symbol in the menu tree and navigating to it can be tedious. To jump
+directly to a symbol, press the :kbd:`/` key. This brings up the following
+dialog, where you can search for symbols by name and jump to them:
 
-#. Press :kbd:`Enter` to return to the options menu.
+.. image:: figures/app_kernel_conf_3.png
+    :align: center
+    :alt: Menuconfig Search Dialog
 
-#. To load any saved kernel configuration file, tab to :guilabel:`< Load >` and
-   press :kbd:`Enter`.
+If you jump to a symbol that isn't currently visible -- e.g. due to having
```

Comment:
better is to write this as:

    .. visible (e.g., due to having unsatisfied dependencies) then ...

Reply:
Got complaints when I added a comma after 'e.g.' in a different project. Maybe I should switch back. ;)

Reply:
Ha. ;)  Most (but not all) writing style guides (e.g., Chicago Manual of Style, The Columbia Guide to Standard American English) recommend using a comma.  If you had used the phrase "for example" a comma would be expected.